.kokoro/client-library-check.sh

```diff
@@ -21,6 +21,52 @@ set -eo pipefail
 # Display commands being run.
 set -x
 
+function get_current_version_from_versions_txt() {
+  versions=$1
+  key=$2
+  version=$(grep "$key:" "${versions}" | cut -d: -f3) # 3rd field is current
+  echo "${version}"
+}
+
+function get_released_version_from_versions_txt() {
+  versions=$1
+  key=$2
+  version=$(grep "$key:" "${versions}" | cut -d: -f2) # 2nd field is release
+  echo "${version}"
+}
+
+function replace_java_shared_config_version() {
+  # replace version
+  xmllint --shell <(cat pom.xml) << EOF
+  setns x=http://maven.apache.org/POM/4.0.0
+  cd .//x:artifactId[text()="google-cloud-shared-config"]
+  cd ../x:version
+  set ${JAVA_SHARED_CONFIG_VERSION}
```

Comment:
nit: from a generic, function design standpoint... using parameters should be preferred over using global state, as it makes the contract of the function when seeing it in use explicit rather than implicit.

In other words:
```shell
replace_java_shared_config_version pom.xml 2.3.4 # I can read this an know exactly what is happening.
```
vs
```shell
replace_java_shared_config_version # I don't know what this affects w/o going to read the function
```

See also "pure functions" and their many benefits.

Reply:
Good point, I've refactored this to accept args instead of using global variables.